### PR TITLE
[PREVIEW] Transpile to dist folder

### PR DESCRIPTION
**Pull request by jozzhart**
Use relative paths in app, fine for test to use root relative paths

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -7,15 +7,15 @@
     "node": ">=8.9.0 <9.0"
   },
   "scripts": {
-    "start": "cross-env NODE_PATH=. node server.js",
+    "start": "node server.js",
```
Comment (davet1985) [inline package.json]: Can we either use `cross-env` or remove it as a dependency? I'm not sure if we actually require it for Azure?

Comment (jozzhart) [reply package.json]: Yeah, not sure what shell Azure uses, and whether cross-env is required for it.  I can try removing it and seeing if the tests still pass if you like?

Comment (davet1985) [reply package.json]: Yeah worth a shot 👍

Comment (jozzhart) [reply package.json]: I guess the other thing is it should be left in for any devs working on Windows?

Comment (davet1985) [reply package.json]: Hmmm maybe

**Pull request merged**